src/Model/Table/AclNodesTable.php

```diff
@@ -185,7 +185,7 @@ public function node($ref = null)
             $query = $this->find('all', $queryData);
 
             if ($query->count() == 0) {
-                throw new Exception\Exception(__d('cake_dev', "AclNode::node() - Couldn't find {0} node identified by \"{1}\"", [$type, print_r($ref, true)]));
+                throw new Exception\Exception(__d('cake_dev', "AclNode::node() - Couldn't find %s node identified by \"%s\"", [$type, print_r($ref, true)]));
```

Comment:
I would have expected the intl style placeholders to work, as the default message formatters are intl based.

Reply:
Didn't work for me with PHP 7.4.3.

Reply:
Have you configured your application to only use the sprintf formatter?

Reply:
To my knowledge we have not set any sprintf formatter configurations explicitly.